packages/test/test-end-to-end-tests/src/test/benchmark/container.memory.spec.ts

```diff
@@ -0,0 +1,130 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { benchmarkMemory, IMemoryTestObject } from "@fluid-tools/benchmark";
+import { IRequest } from "@fluidframework/core-interfaces";
+import {
+    LoaderHeader,
+    IFluidCodeDetails,
+    ILoader,
+    IContainer,
+} from "@fluidframework/container-definitions";
+import {
+    Container,
+    Loader,
+    ILoaderProps,
+} from "@fluidframework/container-loader";
+import {
+    LocalCodeLoader,
+    LoaderContainerTracker,
+    ITestObjectProvider,
+    TestFluidObjectFactory,
+} from "@fluidframework/test-utils";
+import { ensureFluidResolvedUrl } from "@fluidframework/driver-utils";
+import { describeNoCompat } from "@fluidframework/test-version-utils";
+
+const id = "fluid-test://localhost/containerTest";
+const testRequest: IRequest = { url: id };
+const codeDetails: IFluidCodeDetails = { package: "test" };
+
+// REVIEW: enable compat testing?
```

Comment:
I think not having compat for now on either set is fine. I think it's unlikely for time/memory to change with different runtime/loader versions in a way that wasn't existing with homogenous versions of one or the other (put differently, time to do various container operations with runtime@N and loader@M probably aligns with time to do the operation with both at version N or both at version M, and we'd get that historical data with no-compat tests).

Reply:
The comments even came from the existing files in e2e tests which I used as template (the existing, non-perf tests). Removed them here.